### PR TITLE
update zh-trans content/zh/docs/reference/glossary/disruption.md

### PR DESCRIPTION
**Pull request by Sophy417**

### CODE REVIEW

**Comment by tanjunchen** [comment]
/retitle  update zh-trans content/zh/docs/reference/glossary/disruption.md

File: content/zh/docs/reference/glossary/disruption.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: 中断
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/disruption.md]:
```suggestion
title: 干扰
```

File: content/zh/docs/reference/glossary/disruption.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+---
+-->
+<!--
+ Disruptions are events that lead to one or more
+{{< glossary_tooltip term_id="pod" text="Pods" >}} going out of service.
+A disruption has consequences for workload resources, such as
+{{< glossary_tooltip term_id="deployment" >}}, that rely on the affected
+Pods.
+-->
+中断是导致一个或多个事件{{< glossary_tooltip term_id="pod" text="Pods" >}} 停止服务的事件。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/disruption.md]:
```suggestion
干扰是导致一个或多个 {{< glossary_tooltip term_id="pod" text="Pods" >}} 无法提供服务的事件。
```

File: content/zh/docs/reference/glossary/disruption.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+中断会对工作负载资源产生影响，例如{{< glossary_tooltip term_id="deployment" >}}，这取决于受影响的 Pod。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/disruption.md]:
```suggestion
干扰会对依赖于被影响的 Pod 的工作负载资源（例如{{< glossary_tooltip term_id="deployment" >}}）产生影响。
```

File: content/zh/docs/reference/glossary/disruption.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+<!--more-->
+
+<!--
+If you, as cluster operator, destroy a Pod that belongs to an application,
+Kubernetes terms that a _voluntary disruption_. If a Pod goes offline
+because of a Node failure, or an outage affecting a wider failure zone,
+Kubernetes terms that an _involuntary disruption_.
+-->
+如果你作为集群运营商销毁了属于某个应用程序的 Pod， Kubernetes 称之为
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/disruption.md]:
```suggestion
如果你作为集群的运维人员销毁了属于某个应用程序的 Pod， Kubernetes 称之为
```

File: content/zh/docs/reference/glossary/disruption.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+_自愿中断_。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/disruption.md]:
```suggestion
_自愿性干扰_。
```

File: content/zh/docs/reference/glossary/disruption.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+如果由于节点故障或影响更大故障区域的故障使 Pod 离线，Kubernetes 称这是_非自愿中断_。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/disruption.md]:
```suggestion
如果由于节点故障或影响更大故障区域的故障使 Pod 离线，Kubernetes 称这是 _非自愿干扰_。
```

File: content/zh/docs/reference/glossary/disruption.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+<!--
+See [Disruptions](/docs/concepts/workloads/pods/disruptions/) for more information.
+-->
+更多信息，请参看(/docs/concepts/workloads/pods/disruptions/) 。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/disruption.md]:
```suggestion
请参看 [干扰](/docs/concepts/workloads/pods/disruptions/) 以获得更多信息。
```

File: content/zh/docs/reference/glossary/disruption.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+---
+-->
+<!--
+ Disruptions are events that lead to one or more
+{{< glossary_tooltip term_id="pod" text="Pods" >}} going out of service.
+A disruption has consequences for workload resources, such as
+{{< glossary_tooltip term_id="deployment" >}}, that rely on the affected
+Pods.
+-->
+干扰是导致一个或多个 {{< glossary_tooltip term_id="pod" text="Pods" >}} 无法提供服务的事件。
```
Comment (sftim) [inline content/zh/docs/reference/glossary/disruption.md]: nit:
```suggestion
干扰是导致一个或多个 {{< glossary_tooltip term_id="pod" text="Pod" >}} 无法提供服务的事件。
```

**Pull request merged**